### PR TITLE
DIV-6431 service application received

### PR DESCRIPTION
**Pull request by yarpo**
https://tools.hmcts.net/jira/browse/DIV-6431

### CODE REVIEW

File: test/unit/definitions/divorce/UserRoleStateAuth.test.js [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
 
-function byCaseType(caseType) {
+function byCaseType (caseType) {
   return entry => {
     return entry.CaseTypeID === caseType;
   };
 }
 
-function byStateName(stateEntry) {
+function byStateName (stateEntry) {
```
Comment (jacmurphy) [inline test/unit/definitions/divorce/UserRoleStateAuth.test.js:28]: Did you mean to add the space?

File: test/unit/definitions/divorce/UserRoleStateAuth.test.js [hunk 2]
```diff
@@ -1,27 +1,37 @@
...
   return stateAuth => {
     return stateAuth.CaseStateID === stateEntry.ID;
   };
 }
 
-function mapErrorArray(caseType) {
+function mapErrorArray (caseType) {
```
Comment (jacmurphy) [inline test/unit/definitions/divorce/UserRoleStateAuth.test.js:34]: Did you mean to add the space?

Comment (yarpo) [reply test/unit/definitions/divorce/UserRoleStateAuth.test.js:34]: auto-formatting :/ 

It's a very old issue that was caused yearssssss ago by Safari (or even Mozzila). For some reason it was required there and it was causing any issues in other browsers (yes, then Javascript was only used in browsers).

TLDR; It was not needed here.
